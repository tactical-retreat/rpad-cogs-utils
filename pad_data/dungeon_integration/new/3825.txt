Thoth & Sopdet Descended! - Gods of Starry Wisdom-Mythical

stage 0

147 - Keeper of Flame @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

148 - Keeper of Water @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

149 - Keeper of Forest @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

150 - Keeper of Light @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

151 - Keeper of Dark @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

321 - Keeper of Rainbow @ level 1
Basic attacks (2)

stage 1

1212 - Royal Hunting Beast, Griffin @ level 3
Down Burst
Player -99% HP
----- Turn 1 -----
Sniper Eye
Bind 1 random cards for 2~3 turns (1 time use)
----- HP <= 100 -----
Hunting Gale
6521 Damage - Deal 100% damage (5 hits, 20% each)

stage 2

897 - Flame Chaser @ level 12
----- Turn 1 -----
Red Heat
5306 Damage - Change Random to Fire, Deal 80% damage (1 time use)
----- HP <= 100 -----
Flame Blade
6632 Damage - Deal 100% damage (2 hits, 50% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Overheat
10612 Damage - Change Random to Fire, Deal 160% damage

899 - Aqua Chaser @ level 12
----- Turn 1 -----
Hydro Laser
5073 Damage - Change Random to Water, Deal 80% damage (1 time use)
----- HP <= 100 -----
Aqua Lance
6341 Damage - Deal 100% damage (4 hits, 25% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Hydro Burst
10146 Damage - Change Random to Water, Deal 160% damage

901 - Wind Chaser @ level 12
----- Turn 1 -----
Air Slash
5218 Damage - Change Random to Wood, Deal 80% damage (1 time use)
----- HP <= 100 -----
Wind Edge
6523 Damage - Deal 100% damage (5 hits, 20% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Aero Stream
10437 Damage - Change Random to Wood, Deal 160% damage

903 - Lightning Chaser @ level 12
----- Turn 1 -----
Discharge
5073 Damage - Change Random to Light, Deal 80% damage (1 time use)
----- HP <= 100 -----
Volt Axe
6975 Damage - Deal 110% damage
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Thunder Weapon
10146 Damage - Change Random to Light, Deal 160% damage

905 - Phantom Chaser @ level 12
----- Turn 1 -----
Fear Attack
5073 Damage - Change Random to Dark, Deal 80% damage (1 time use)
----- HP <= 100 -----
Quad Size
6341 Damage - Deal 100% damage (4 hits, 25% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Shadow Mist
10146 Damage - Change Random to Dark, Deal 160% damage

stage 3

998 - Dark Sun Deity, Ra @ level 3
Protection of the Gods
Voids status ailments for 999 turns
----- Turn 1 -----
Ray of the Sun
Fire skyfall +10% for 99 turns
----- Turn 2 -----
Solar Beam
777 Damage - Deal 10% damage
----- Turn 3 -----
Soler Lazer
7777 Damage - Deal 100% damage
----- Turn 4 -----
Solar Cannon
15554 Damage - Deal 200% damage
----- HP <= 100 -----
Solar Explosion
77770 Damage - Deal 1000% damage

stage 4

997 - Shining Sea Deity, Isis @ level 3
Young Firebird's Ambush
2747 Damage - Deal 50% damage
----- HP <= 100 -----
Holy Ocean's Wave
4395 Damage - Change Wood to Water, Deal 80% damage (50% chance)
Burning Life Light
4395 Damage - Change Dark to Light, Deal 80% damage (80% chance)
----- HP <= 50 -----
The Flooding of the Nile
Bind active skills for 5 turns (1 time use)
Star Mother & Firebird's Flash
8790 Damage - Deal 160% damage (2 hits, 80% each)

stage 5

1462 - Sopdet @ level 3
Grace of the Nile
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

1464 - Thoth @ level 3
Hieroglyphic Carvings
Voids status ailments for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

stage 0

147 - Keeper of Flame @ level 3
Preemptive
16878 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
45009 Damage - Deal 200% damage

148 - Keeper of Water @ level 3
Preemptive
16878 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
45009 Damage - Deal 200% damage

149 - Keeper of Forest @ level 3
Preemptive
16878 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
45009 Damage - Deal 200% damage

151 - Keeper of Dark @ level 3
Preemptive
16878 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
45009 Damage - Deal 200% damage

321 - Keeper of Rainbow @ level 2
----- HP <= 100 -----
Lance Attacks
64870 Damage - Deal 240% damage (3 hits, 80% each)

stage 1

1212 - Royal Hunting Beast, Griffin @ level 5
Down Burst
Player -99% HP
----- Turn 1 -----
Sniper Eye
Bind 1 random cards for 2~3 turns (1 time use)
----- HP <= 100 -----
Hunting Gale
12166 Damage - Deal 100% damage (5 hits, 20% each)

stage 2

899 - Aqua Chaser @ level 14
----- Turn 1 -----
Hydro Laser
5970 Damage - Change Random to Water, Deal 80% damage (1 time use)
----- HP <= 100 -----
Aqua Lance
7462 Damage - Deal 100% damage (4 hits, 25% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Hydro Burst
11940 Damage - Change Random to Water, Deal 160% damage

901 - Wind Chaser @ level 14
----- Turn 1 -----
Air Slash
6141 Damage - Change Random to Wood, Deal 80% damage (1 time use)
----- HP <= 100 -----
Wind Edge
7677 Damage - Deal 100% damage (5 hits, 20% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Aero Stream
12283 Damage - Change Random to Wood, Deal 160% damage

903 - Lightning Chaser @ level 14
----- Turn 1 -----
Discharge
5970 Damage - Change Random to Light, Deal 80% damage (1 time use)
----- HP <= 100 -----
Volt Axe
8208 Damage - Deal 110% damage
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Thunder Weapon
11940 Damage - Change Random to Light, Deal 160% damage

905 - Phantom Chaser @ level 14
----- Turn 1 -----
Fear Attack
5970 Damage - Change Random to Dark, Deal 80% damage (1 time use)
----- HP <= 100 -----
Quad Size
7462 Damage - Deal 100% damage (4 hits, 25% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Shadow Mist
11940 Damage - Change Random to Dark, Deal 160% damage

stage 3

998 - Dark Sun Deity, Ra @ level 5
Protection of the Gods
Voids status ailments for 999 turns
----- Turn 1 -----
Ray of the Sun
Fire skyfall +10% for 99 turns
----- Turn 2 -----
Solar Beam
777 Damage - Deal 10% damage
----- Turn 3 -----
Soler Lazer
7777 Damage - Deal 100% damage
----- Turn 4 -----
Solar Cannon
15554 Damage - Deal 200% damage
----- HP <= 100 -----
Solar Explosion
77770 Damage - Deal 1000% damage

stage 4

997 - Shining Sea Deity, Isis @ level 5
Young Firebird's Ambush
5125 Damage - Deal 50% damage
----- HP <= 100 -----
Holy Ocean's Wave
8200 Damage - Change Wood to Water, Deal 80% damage (50% chance)
Burning Life Light
8200 Damage - Change Dark to Light, Deal 80% damage (80% chance)
----- HP <= 50 -----
The Flooding of the Nile
Bind active skills for 5 turns (1 time use)
Star Mother & Firebird's Flash
16400 Damage - Deal 160% damage (2 hits, 80% each)

stage 5

1462 - Sopdet @ level 5
Grace of the Nile
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

1464 - Thoth @ level 5
Hieroglyphic Carvings
Voids status ailments for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

Thoth & Sopdet Descended! - Gods of Starry Wisdom-Legend

stage 0

147 - Keeper of Flame @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

148 - Keeper of Water @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

149 - Keeper of Forest @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

150 - Keeper of Light @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

151 - Keeper of Dark @ level 2
Preemptive
9572 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
25527 Damage - Deal 200% damage

321 - Keeper of Rainbow @ level 1
Basic attacks (2)

stage 1

1212 - Royal Hunting Beast, Griffin @ level 3
Down Burst
Player -99% HP
----- Turn 1 -----
Sniper Eye
Bind 1 random cards for 2~3 turns (1 time use)
----- HP <= 100 -----
Hunting Gale
6521 Damage - Deal 100% damage (5 hits, 20% each)

stage 2

897 - Flame Chaser @ level 12
----- Turn 1 -----
Red Heat
5306 Damage - Change Random to Fire, Deal 80% damage (1 time use)
----- HP <= 100 -----
Flame Blade
6632 Damage - Deal 100% damage (2 hits, 50% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Overheat
10612 Damage - Change Random to Fire, Deal 160% damage

899 - Aqua Chaser @ level 12
----- Turn 1 -----
Hydro Laser
5073 Damage - Change Random to Water, Deal 80% damage (1 time use)
----- HP <= 100 -----
Aqua Lance
6341 Damage - Deal 100% damage (4 hits, 25% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Hydro Burst
10146 Damage - Change Random to Water, Deal 160% damage

901 - Wind Chaser @ level 12
----- Turn 1 -----
Air Slash
5218 Damage - Change Random to Wood, Deal 80% damage (1 time use)
----- HP <= 100 -----
Wind Edge
6523 Damage - Deal 100% damage (5 hits, 20% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Aero Stream
10437 Damage - Change Random to Wood, Deal 160% damage

903 - Lightning Chaser @ level 12
----- Turn 1 -----
Discharge
5073 Damage - Change Random to Light, Deal 80% damage (1 time use)
----- HP <= 100 -----
Volt Axe
6975 Damage - Deal 110% damage
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Thunder Weapon
10146 Damage - Change Random to Light, Deal 160% damage

905 - Phantom Chaser @ level 12
----- Turn 1 -----
Fear Attack
5073 Damage - Change Random to Dark, Deal 80% damage (1 time use)
----- HP <= 100 -----
Quad Size
6341 Damage - Deal 100% damage (4 hits, 25% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Shadow Mist
10146 Damage - Change Random to Dark, Deal 160% damage

stage 3

998 - Dark Sun Deity, Ra @ level 3
Protection of the Gods
Voids status ailments for 999 turns
----- Turn 1 -----
Ray of the Sun
Fire skyfall +10% for 99 turns
----- Turn 2 -----
Solar Beam
777 Damage - Deal 10% damage
----- Turn 3 -----
Soler Lazer
7777 Damage - Deal 100% damage
----- Turn 4 -----
Solar Cannon
15554 Damage - Deal 200% damage
----- HP <= 100 -----
Solar Explosion
77770 Damage - Deal 1000% damage

stage 4

997 - Shining Sea Deity, Isis @ level 3
Young Firebird's Ambush
2747 Damage - Deal 50% damage
----- HP <= 100 -----
Holy Ocean's Wave
4395 Damage - Change Wood to Water, Deal 80% damage (50% chance)
Burning Life Light
4395 Damage - Change Dark to Light, Deal 80% damage (80% chance)
----- HP <= 50 -----
The Flooding of the Nile
Bind active skills for 5 turns (1 time use)
Star Mother & Firebird's Flash
8790 Damage - Deal 160% damage (2 hits, 80% each)

stage 5

1462 - Sopdet @ level 3
Grace of the Nile
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

1464 - Thoth @ level 3
Hieroglyphic Carvings
Voids status ailments for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

stage 0

147 - Keeper of Flame @ level 3
Preemptive
16878 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
45009 Damage - Deal 200% damage

148 - Keeper of Water @ level 3
Preemptive
16878 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
45009 Damage - Deal 200% damage

149 - Keeper of Forest @ level 3
Preemptive
16878 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
45009 Damage - Deal 200% damage

151 - Keeper of Dark @ level 3
Preemptive
16878 Damage - Deal 75% damage
----- HP <= 25 -----
Big Lance
45009 Damage - Deal 200% damage

321 - Keeper of Rainbow @ level 2
----- HP <= 100 -----
Lance Attacks
64870 Damage - Deal 240% damage (3 hits, 80% each)

stage 1

1212 - Royal Hunting Beast, Griffin @ level 5
Down Burst
Player -99% HP
----- Turn 1 -----
Sniper Eye
Bind 1 random cards for 2~3 turns (1 time use)
----- HP <= 100 -----
Hunting Gale
12166 Damage - Deal 100% damage (5 hits, 20% each)

stage 2

899 - Aqua Chaser @ level 14
----- Turn 1 -----
Hydro Laser
5970 Damage - Change Random to Water, Deal 80% damage (1 time use)
----- HP <= 100 -----
Aqua Lance
7462 Damage - Deal 100% damage (4 hits, 25% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Hydro Burst
11940 Damage - Change Random to Water, Deal 160% damage

901 - Wind Chaser @ level 14
----- Turn 1 -----
Air Slash
6141 Damage - Change Random to Wood, Deal 80% damage (1 time use)
----- HP <= 100 -----
Wind Edge
7677 Damage - Deal 100% damage (5 hits, 20% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Aero Stream
12283 Damage - Change Random to Wood, Deal 160% damage

903 - Lightning Chaser @ level 14
----- Turn 1 -----
Discharge
5970 Damage - Change Random to Light, Deal 80% damage (1 time use)
----- HP <= 100 -----
Volt Axe
8208 Damage - Deal 110% damage
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Thunder Weapon
11940 Damage - Change Random to Light, Deal 160% damage

905 - Phantom Chaser @ level 14
----- Turn 1 -----
Fear Attack
5970 Damage - Change Random to Dark, Deal 80% damage (1 time use)
----- HP <= 100 -----
Quad Size
7462 Damage - Deal 100% damage (4 hits, 25% each)
----- HP <= 50 -----
Chase Mode
Increase damage to 200% for the next 5 turns
----- HP <= 24 -----
Shadow Mist
11940 Damage - Change Random to Dark, Deal 160% damage

stage 3

998 - Dark Sun Deity, Ra @ level 5
Protection of the Gods
Voids status ailments for 999 turns
----- Turn 1 -----
Ray of the Sun
Fire skyfall +10% for 99 turns
----- Turn 2 -----
Solar Beam
777 Damage - Deal 10% damage
----- Turn 3 -----
Soler Lazer
7777 Damage - Deal 100% damage
----- Turn 4 -----
Solar Cannon
15554 Damage - Deal 200% damage
----- HP <= 100 -----
Solar Explosion
77770 Damage - Deal 1000% damage

stage 4

997 - Shining Sea Deity, Isis @ level 5
Young Firebird's Ambush
5125 Damage - Deal 50% damage
----- HP <= 100 -----
Holy Ocean's Wave
8200 Damage - Change Wood to Water, Deal 80% damage (50% chance)
Burning Life Light
8200 Damage - Change Dark to Light, Deal 80% damage (80% chance)
----- HP <= 50 -----
The Flooding of the Nile
Bind active skills for 5 turns (1 time use)
Star Mother & Firebird's Flash
16400 Damage - Deal 160% damage (2 hits, 80% each)

stage 5

1462 - Sopdet @ level 5
Grace of the Nile
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

1464 - Thoth @ level 5
Hieroglyphic Carvings
Voids status ailments for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

