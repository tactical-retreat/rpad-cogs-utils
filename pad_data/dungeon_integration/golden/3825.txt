Thoth & Sopdet Descended! - Gods of Starry Wisdom-Mythical

stage 0

147 - Keeper of Flame @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

149 - Keeper of Forest @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

148 - Keeper of Water @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

150 - Keeper of Light @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

151 - Keeper of Dark @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

321 - Keeper of Rainbow @ level 1
Basic attacks (2)

stage 1

1212 - Royal Hunting Beast, Griffin @ level 3
Down Burst
Player -99% HP
----- Turn 1 -----
Sniper Eye
Bind 1 random cards for 2~3 turns
----- HP <= 100 -----
Hunting Gale
6521 Damage - Deal 100% damage (5 hits, 20% each)

stage 2

897 - Flame Chaser @ level 12
----- HP <= 100 -----
Flame Blade
6632 Damage - Deal 100% damage (2 hits, 50% each)

899 - Aqua Chaser @ level 12
----- HP <= 100 -----
Aqua Lance
6341 Damage - Deal 100% damage (4 hits, 25% each)

901 - Wind Chaser @ level 12
----- HP <= 100 -----
Wind Edge
6523 Damage - Deal 100% damage (5 hits, 20% each)

905 - Phantom Chaser @ level 12
----- HP <= 100 -----
Quad Size
6341 Damage - Deal 100% damage (4 hits, 25% each)

903 - Lightning Chaser @ level 12
----- HP <= 100 -----
Volt Axe
6975 Damage - Deal 110% damage

stage 3

998 - Dark Sun Deity, Ra @ level 3
Protection of the Gods
Voids status ailments for 999 turns
----- Turn 1 -----
Ray of the Sun
Fire skyfall +10% for 99 turns
----- Turn 2 -----
Solar Beam
777 Damage - Deal 10% damage
----- Turn 3 -----
Soler Lazer
7777 Damage - Deal 100% damage
----- Turn 4 -----
Solar Cannon
15554 Damage - Deal 200% damage
----- HP <= 100 -----
Solar Explosion
77770 Damage - Deal 1000% damage

stage 4

997 - Shining Sea Deity, Isis @ level 3
Young Firebird's Ambush
2747 Damage - Deal 50% damage
----- HP <= 100 -----
Holy Ocean's Wave
4395 Damage - Change Wood to Water, Deal 80% damage
Burning Life Light
4395 Damage - Change Dark to Light, Deal 80% damage
----- HP <= 50 -----
The Flooding of the Nile
Bind active skills for 5 turns
Star Mother & Firebird's Flash
8790 Damage - Deal 160% damage (2 hits, 80% each)
----- HP <= 50 -----
Star Mother & Firebird's Flash
8790 Damage - Deal 160% damage (2 hits, 80% each)

stage 5

1462 - Sopdet @ level 3
Grace of the Nile
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

1464 - Thoth @ level 3
Hieroglyphic Carvings
Voids status ailments for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

stage 0

147 - Keeper of Flame @ level 3
----- HP <= 100 -----
Preemptive
16878 Damage - Deal 75% damage

148 - Keeper of Water @ level 3
----- HP <= 100 -----
Preemptive
16878 Damage - Deal 75% damage

149 - Keeper of Forest @ level 3
----- HP <= 100 -----
Preemptive
16878 Damage - Deal 75% damage

151 - Keeper of Dark @ level 3
----- HP <= 100 -----
Preemptive
16878 Damage - Deal 75% damage

321 - Keeper of Rainbow @ level 2
----- HP <= 100 -----
Lance Attacks
64870 Damage - Deal 240% damage (3 hits, 80% each)

stage 1

1212 - Royal Hunting Beast, Griffin @ level 5
Down Burst
Player -99% HP
----- Turn 1 -----
Sniper Eye
Bind 1 random cards for 2~3 turns
----- HP <= 100 -----
Hunting Gale
12166 Damage - Deal 100% damage (5 hits, 20% each)

stage 2

899 - Aqua Chaser @ level 14
----- HP <= 100 -----
Aqua Lance
7462 Damage - Deal 100% damage (4 hits, 25% each)

901 - Wind Chaser @ level 14
----- HP <= 100 -----
Wind Edge
7677 Damage - Deal 100% damage (5 hits, 20% each)

903 - Lightning Chaser @ level 14
----- HP <= 100 -----
Volt Axe
8208 Damage - Deal 110% damage

905 - Phantom Chaser @ level 14
----- HP <= 100 -----
Quad Size
7462 Damage - Deal 100% damage (4 hits, 25% each)

stage 3

998 - Dark Sun Deity, Ra @ level 5
Protection of the Gods
Voids status ailments for 999 turns
----- Turn 1 -----
Ray of the Sun
Fire skyfall +10% for 99 turns
----- Turn 2 -----
Solar Beam
777 Damage - Deal 10% damage
----- Turn 3 -----
Soler Lazer
7777 Damage - Deal 100% damage
----- Turn 4 -----
Solar Cannon
15554 Damage - Deal 200% damage
----- HP <= 100 -----
Solar Explosion
77770 Damage - Deal 1000% damage

stage 4

997 - Shining Sea Deity, Isis @ level 5
Young Firebird's Ambush
5125 Damage - Deal 50% damage
----- HP <= 100 -----
Holy Ocean's Wave
8200 Damage - Change Wood to Water, Deal 80% damage
Burning Life Light
8200 Damage - Change Dark to Light, Deal 80% damage
----- HP <= 50 -----
The Flooding of the Nile
Bind active skills for 5 turns
Star Mother & Firebird's Flash
16400 Damage - Deal 160% damage (2 hits, 80% each)
----- HP <= 50 -----
Star Mother & Firebird's Flash
16400 Damage - Deal 160% damage (2 hits, 80% each)

stage 5

1462 - Sopdet @ level 5
Grace of the Nile
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

1464 - Thoth @ level 5
Hieroglyphic Carvings
Voids status ailments for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

Thoth & Sopdet Descended! - Gods of Starry Wisdom-Legend

stage 0

147 - Keeper of Flame @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

149 - Keeper of Forest @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

148 - Keeper of Water @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

150 - Keeper of Light @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

151 - Keeper of Dark @ level 2
----- HP <= 100 -----
Preemptive
9572 Damage - Deal 75% damage

321 - Keeper of Rainbow @ level 1
Basic attacks (2)

stage 1

1212 - Royal Hunting Beast, Griffin @ level 3
Down Burst
Player -99% HP
----- Turn 1 -----
Sniper Eye
Bind 1 random cards for 2~3 turns
----- HP <= 100 -----
Hunting Gale
6521 Damage - Deal 100% damage (5 hits, 20% each)

stage 2

897 - Flame Chaser @ level 12
----- HP <= 100 -----
Flame Blade
6632 Damage - Deal 100% damage (2 hits, 50% each)

899 - Aqua Chaser @ level 12
----- HP <= 100 -----
Aqua Lance
6341 Damage - Deal 100% damage (4 hits, 25% each)

901 - Wind Chaser @ level 12
----- HP <= 100 -----
Wind Edge
6523 Damage - Deal 100% damage (5 hits, 20% each)

905 - Phantom Chaser @ level 12
----- HP <= 100 -----
Quad Size
6341 Damage - Deal 100% damage (4 hits, 25% each)

903 - Lightning Chaser @ level 12
----- HP <= 100 -----
Volt Axe
6975 Damage - Deal 110% damage

stage 3

998 - Dark Sun Deity, Ra @ level 3
Protection of the Gods
Voids status ailments for 999 turns
----- Turn 1 -----
Ray of the Sun
Fire skyfall +10% for 99 turns
----- Turn 2 -----
Solar Beam
777 Damage - Deal 10% damage
----- Turn 3 -----
Soler Lazer
7777 Damage - Deal 100% damage
----- Turn 4 -----
Solar Cannon
15554 Damage - Deal 200% damage
----- HP <= 100 -----
Solar Explosion
77770 Damage - Deal 1000% damage

stage 4

997 - Shining Sea Deity, Isis @ level 3
Young Firebird's Ambush
2747 Damage - Deal 50% damage
----- HP <= 100 -----
Holy Ocean's Wave
4395 Damage - Change Wood to Water, Deal 80% damage
Burning Life Light
4395 Damage - Change Dark to Light, Deal 80% damage
----- HP <= 50 -----
The Flooding of the Nile
Bind active skills for 5 turns
Star Mother & Firebird's Flash
8790 Damage - Deal 160% damage (2 hits, 80% each)
----- HP <= 50 -----
Star Mother & Firebird's Flash
8790 Damage - Deal 160% damage (2 hits, 80% each)

stage 5

1462 - Sopdet @ level 3
Grace of the Nile
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

1464 - Thoth @ level 3
Hieroglyphic Carvings
Voids status ailments for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

stage 0

147 - Keeper of Flame @ level 3
----- HP <= 100 -----
Preemptive
16878 Damage - Deal 75% damage

148 - Keeper of Water @ level 3
----- HP <= 100 -----
Preemptive
16878 Damage - Deal 75% damage

149 - Keeper of Forest @ level 3
----- HP <= 100 -----
Preemptive
16878 Damage - Deal 75% damage

151 - Keeper of Dark @ level 3
----- HP <= 100 -----
Preemptive
16878 Damage - Deal 75% damage

321 - Keeper of Rainbow @ level 2
----- HP <= 100 -----
Lance Attacks
64870 Damage - Deal 240% damage (3 hits, 80% each)

stage 1

1212 - Royal Hunting Beast, Griffin @ level 5
Down Burst
Player -99% HP
----- Turn 1 -----
Sniper Eye
Bind 1 random cards for 2~3 turns
----- HP <= 100 -----
Hunting Gale
12166 Damage - Deal 100% damage (5 hits, 20% each)

stage 2

899 - Aqua Chaser @ level 14
----- HP <= 100 -----
Aqua Lance
7462 Damage - Deal 100% damage (4 hits, 25% each)

901 - Wind Chaser @ level 14
----- HP <= 100 -----
Wind Edge
7677 Damage - Deal 100% damage (5 hits, 20% each)

903 - Lightning Chaser @ level 14
----- HP <= 100 -----
Volt Axe
8208 Damage - Deal 110% damage

905 - Phantom Chaser @ level 14
----- HP <= 100 -----
Quad Size
7462 Damage - Deal 100% damage (4 hits, 25% each)

stage 3

998 - Dark Sun Deity, Ra @ level 5
Protection of the Gods
Voids status ailments for 999 turns
----- Turn 1 -----
Ray of the Sun
Fire skyfall +10% for 99 turns
----- Turn 2 -----
Solar Beam
777 Damage - Deal 10% damage
----- Turn 3 -----
Soler Lazer
7777 Damage - Deal 100% damage
----- Turn 4 -----
Solar Cannon
15554 Damage - Deal 200% damage
----- HP <= 100 -----
Solar Explosion
77770 Damage - Deal 1000% damage

stage 4

997 - Shining Sea Deity, Isis @ level 5
Young Firebird's Ambush
5125 Damage - Deal 50% damage
----- HP <= 100 -----
Holy Ocean's Wave
8200 Damage - Change Wood to Water, Deal 80% damage
Burning Life Light
8200 Damage - Change Dark to Light, Deal 80% damage
----- HP <= 50 -----
The Flooding of the Nile
Bind active skills for 5 turns
Star Mother & Firebird's Flash
16400 Damage - Deal 160% damage (2 hits, 80% each)
----- HP <= 50 -----
Star Mother & Firebird's Flash
16400 Damage - Deal 160% damage (2 hits, 80% each)

stage 5

1462 - Sopdet @ level 5
Grace of the Nile
Absorb combo <= 4 damage for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

1464 - Thoth @ level 5
Hieroglyphic Carvings
Voids status ailments for 999 turns
----- HP <= 100 -----
Ceremony of Rebirth
Enemy ally recover 50% hp

